department.rb

```diff
@@ -0,0 +1,48 @@
+require './employee.rb'
+
+
+class Department
+
+  def initialize(department_name)
+    @department_name = department_name
+    @employees = []
+  end
+
+  def department_name
+    @department_name
+  end
+
+  def add_employee(employee)
+    @employees << employee
+  end
+
+  def employees
+    @employees
+  end
+
+  def payroll
+    total = 0
+    @employees.each do |employee|
+      total += employee.salary
+    end
+    return total
+  end
+
+
+
+  # def hr
```

Comment:
It's best to not commit commented code like this.  When you have code that was working and you want to have access to it later, you should be able to access it using the git history rather than keeping it in your active files.